app/controllers/Whereuat.scala

```diff
@@ -70,6 +70,14 @@ class Whereuat extends Controller {
 
 
   // Route actions
+  def index = Action {
+    Ok(views.html.index())
+  }
+
+  def team = Action {
```

Comment:
it's just easier this way i think. don't have to rely on angular or something and it's super simple